README.md

```diff
@@ -27,3 +27,11 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+An invariant for this implementation of a fibonacci function would be that for any natural number n, the fibonacci function will return the nth fibonacci number, and all fibonacci values up to n that have already been computed will be used to correctly compute the next values up to n. This invariant helps us prove the function correct by stating that each recursive call correctly computes the nth fibonacci number by using the previously correctly computed values. The first two previously computed values are the base cases of 1 and 0, which we know the fib values of. Fib(2) will be computed using the correctly computed base cases so that each value after that will be correctly computed using the two previous values stored in the array.
```

Comment:
But what is true at the beginning of the recursive call, i.e. before you're computing the current Fibonacci number?